integration-web/src/main/java/org/symphonyoss/integration/web/resource/ApplicationAuthorizationResource.java

```diff
@@ -145,21 +145,14 @@ public ResponseEntity getUserAuthorizationData(@PathVariable String configuratio
    */
   @RequestMapping(value = "/authorize")
   public ResponseEntity authorize(@PathVariable String configurationId, HttpServletRequest request,
-      @RequestBody(required = false) String body) throws RemoteApiException {
+      @RequestBody(required = false) String body) throws RemoteApiException,
+      AuthorizationException {
```

Comment:
Why don't you encapsulate this AuthorizationException in a RemoteApiException? I believe our endpoints should throw only RemoteApiException.